test/unit/lib/gpHelper.js

```diff
@@ -98,4 +98,42 @@ describe('gp counts helper', () => {
       expect(message[0]).to.equal('1 GP');
     });
   });
+
+  describe('getPersonSingular', () => {
+    it('should return \'are\' for more than one male and female gp', () => {
```

Comment:
I've changed the GP counts to both be 2, but left the text as 'more than one' rather than several so it is consistently worded with the pluralise tests.